### PR TITLE
Test coverage

### PR DESCRIPTION
**Pull request by dpsanders**

### CODE REVIEW

**Comment by dpsanders** [comment]
This does some clean up related to tests. In particular, autodiff was not being used in the root finding tests. Some code was rewritten to note give misleading test results. Ready when green.

**Comment by dpsanders** [comment]
I am planning to tag 0.1.2 once this is merged.

File: test/interval_tests/consistency_tests.jl [hunk 1]
```diff
@@ -83,13 +83,20 @@ facts("Precision tests") do
...
+
 facts("Constructing intervals") do
 
-    set_interval_precision(64)
+    set_interval_precision(Float64)
     a = @interval("[0.1, 0.2]")
     b = @interval(0.1, 0.2)
-    #@show a,b
-    @fact a == b --> true
+    @show a,b
```
Comment (lbenet) [inline test/interval_tests/consistency_tests.jl]: Is this `@show` needed?

File: test/root_finding_tests/root_finding_tests.jl [hunk 2]
```diff
@@ -30,7 +30,8 @@ big_pi = @interval(pi)
 set_interval_precision(Float64)
 float_pi = @interval(pi)
 
-Base.⊆(a::Interval, b::Root) = a ⊆ b[1]   # the Root object has the interval in the first entry
+Base.⊆(a::Interval, b::Root) = a ⊆ b.interval   # the Root object has the interval in the first entry
```
Comment (lbenet) [inline test/root_finding_tests/root_finding_tests.jl]: I think this functions should be defined in the package itself. I propose to move them to `root_finding/root_finding.jl`

File: test/interval_tests/consistency_tests.jl (hunk 1 above)
Comment (dpsanders) [inline test/interval_tests/consistency_tests.jl]: Nope...

File: test/root_finding_tests/root_finding_tests.jl (hunk 2 above)
Comment (dpsanders) [inline test/root_finding_tests/root_finding_tests.jl]: OK, good idea.

Comment (lbenet) [inline test/root_finding_tests/root_finding_tests.jl]: I wait to your update, and then I merge this. Really nice!

**Comment by lbenet** [comment]
Merged, and ready for v 0.1.2

**Pull request merged**